testsuite/tests/ui/search/test_account_search.py

```diff
@@ -58,20 +73,18 @@ def test_search_multiple_accounts(login, navigator, custom_account, request):
         - you search account by specific name it will return the correct one
     """
     name = blame(request, "")
```

Comment:
Aren't you missing the first string or are all names supposed to start with a dash?